phpunit/blocks/render-block-media-text-test.php

```diff
@@ -81,15 +81,15 @@ public function test_render_block_core_media_text_featured_image() {
 		$rendered   = gutenberg_render_block_core_media_text( $attributes, $content );
 		$this->assertStringContainsString( '<img alt="" src="' . wp_get_attachment_image_url( self::$attachment_id, 'full' ) . '"', $rendered );
 
-		// Assert that the rendered block contains the featured image as the background-image url,
-		// and not the image element, when image fill is true.
+		// Assert that the rendered block contains the featured image as an image element
+		// and not as a background-image url, when image fill is true.
 		$attributes = array(
 			'useFeaturedImage' => true,
 			'imageFill'        => true,
 		);
 		$rendered   = gutenberg_render_block_core_media_text( $attributes, $content );
-		$this->assertStringContainsString( 'background-image:url(' . wp_get_attachment_image_url( self::$attachment_id, 'full' ) . ')', $rendered );
-		$this->assertStringNotContainsString( '<img', $rendered );
+		$this->assertStringNotContainsString( 'background-image:', $rendered );
```

Comment:
Nit: I think we can remove this `notContains` check..

Reply:
Done in f11a6b3cdf8aac4155ae1eb64ddff29faf1de559.